src/NLog/Layouts/Layout.cs

```diff
@@ -54,7 +54,7 @@ public abstract class Layout : ISupportsInitialize, IRenderable
         /// Is this layout initialized? See <see cref="Initialize(NLog.Config.LoggingConfiguration)"/>
         /// </summary>
         internal bool IsInitialized;
-        private bool _scannedForObjects;
+        internal bool _scannedForObjects;
```

Comment:
@snakefoot why is internal needed?

Reply:
Yes that was just me trying to skip the scanning of object-graph when seeing that it was "fixed"-value. Will improve on this, so the optimization will be true.